release.sh

```diff
@@ -1,16 +1,11 @@
 #!/bin/sh
-#
+
 VERSION=`node -p "require('./package.json').version"`
 DATE_STR="$(date +%F)"
-CURRENT_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
-
-if [[ ! ${npm_config_git_tag_version+tag} ]]; then
-   echo "Tagging $CURRENT_BRANCH with version $VERSION ($DATE_STR)\n"
-fi
 
 # Update CHANGELOG.md with current version and date if this is not a release
 # candidate version bump.
-if [[ ! $VERSION =~ - ]]; then
+if [ -n $VERSION ]; then
```

Comment:
This check is POSIX-compliant; basically just checks that version is "not empty".